astropy/coordinates/tests/test_funcs.py

```diff
@@ -49,3 +51,20 @@ def test_concatenate():
 
     with pytest.raises(TypeError):
         concatenate(1*u.deg)
+
+
+def test_constellations():
+    from .. import ICRS, FK5
+    from ..funcs import get_constellation
+
+    inuma = ICRS(9*u.hour, 65*u.deg)
```

Comment:
Might be worth including a specific test for Boötes, it being the unicode oddball.